i18n/ko.json

```diff
@@ -0,0 +1,111 @@
+{
+  "install-npm": {
+    "verifying": "npm이 설치되어있는지 확인하고 있습니다...",
+    "missing": "npm이 설치되어있지 않은 것 같습니다.",
+    "npm-problem": "이런! npm명령어 실행 에러가 발생했습니다.",
+    "version-verified": "현재 {{version}} 버전을 사용하고 있습니다. 멋져요!\n그럼 최신 버전을 확인해 보겠습니다. 잠시 기다려주세요...",
+    "latest-version": "최신 버전은 {{version}} 입니다.",
```

Comment:
If the `{{version}}` won't be replaced when there are no surrounding space, then it's okay to leave the blank space.